### DISCUSSION TITLE
怎么动态创建一个layer ，然后所有的entity 加到这个layer之后 ，对这个layer 进行显示隐藏

### DISCUSSION THREAD

**Discussion by YinMuHuaDao** [zouyaoji/vue-cesium#575] [Q&A]
怎么动态创建一个layer ，然后所有的entity 加到这个layer之后 ，对这个layer 进行显示隐藏

**Comment by zouyaoji** [comment]
cesium里面。实体没有layer的概念，叫datasource，你这个需求可以看看https://zouyaoji.top/vue-cesium/#/en-US/component/datasources/vc-datasource-custom

**Comment by YinMuHuaDao** [comment]
十分感谢

**Comment by YinMuHuaDao** [comment]
就是再问下datasource里面加载的都是cesium里面的实体 ， 怎么添加vue-cesium里面封装好了的实体呢

**Reply by zouyaoji** [reply]
参考文档哈 有例子

**Comment by YinMuHuaDao** [comment]
好的谢谢
